src/tikzDevice.c

```diff
@@ -315,6 +317,7 @@ static Rboolean TikZ_Setup(
   tikzInfo->onefile = onefile;
   tikzInfo->timestamp = timestamp;
   tikzInfo->verbose = verbose;
+  tikzInfo->pictureOptions = calloc_strcpy(pictureOptions);
```

Comment:
Do we need to free this after we're done?

Reply:
We might need a call to `free()` that corresponds to the `calloc_strcpy()` call.